docs/howto/deploy/index.md

```diff
@@ -14,9 +15,29 @@ redirect_from:
 
 # Deploy and Setup lakeFS
 
-For a hosted lakeFS service with guaranteed SLAs, try [lakeFS Cloud](https://lakefs.cloud)
-{: .note }
+{: .tip }
+> The instructions given here are for a self-managed deployment of lakeFS. 
+> 
+> For a hosted lakeFS service with guaranteed SLAs, try [lakeFS Cloud](https://lakefs.cloud)
+
+lakeFS has several dependencies for which you need to select and configure a technology or interface. 
+
+* Object Store
+* Authentication
+* Key-Value Store
+* Execution Platform
+
```

Comment:
This phrasing mixes stuff that shouldn't be mixed. The word "dependencies" here is confusing.

Object store: well, yes.
Authentication: what does it mean? why is it a prerequisite? 
Key-value store: not a prerequisite but how lakeFS will use your database. The prerequisite is a database.
Execution platform: not a prerequisite. it goes without saying. there are other ways in which you can mention that.

Let's respect our users' time.

```
This section will guide you through deploying lakeFS on your cloud, on top of an object store. This will require a database.
```

Reply:
You make a good point. I've pushed a reworded version of this.